files/build-kernel.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -e
+
+MAKEOPTS="-j$(( $(getconf _NPROCESSORS_ONLN) + 1 ))"
```

Comment:
you might want to control load with `-l [load], --load-average[=load]
            Specifies that no new jobs (commands) should be started if there are others jobs running and the load average is at least load (a floating-point number).  With no argument, removes a previous load limit.`